src/components/incident/ManualCloseSignOffAction.tsx

```diff
@@ -29,37 +30,60 @@ const ManualClose: React.FC<ManualClosePropsType> = ({
   signOffActionProps = {},
   reopenButtonProps = {},
   ButtonComponent = Button,
+  isBulk,
 }: ManualClosePropsType) => {
   const classes = useStyles();
 
+  const signOffActionDefaultProps = {
+    dialogTitle: "Manually close incident",
+    dialogContentText: "Write a message describing why the incident was manually closed",
+    dialogSubmitText:"Close now",
+    dialogCancelText:"Cancel",
+    dialogButtonText: closeButtonText,
+    dialogInputLabel: "Closing message",
+    isDialogInputRequired: false,
+    dialogInputType: "text",
+    title: "Manually close incident",
+    question: "Are you sure you want to close this incident?",
+    onSubmit: onManualClose,
+  }
+
+  const signOffActionPluralProps = {
+    dialogTitle: "Manually close incidents",
+    dialogContentText: "Write a message describing why the incidents were manually closed",
+    question: "Are you sure you want to close these incidents?",
+  }
+
+  const reopenButtonDefaultProps = {
+    title: "Reopen incident",
+    question: "Are you sure you want to reopen this incident?",
+    onConfirm: onManualOpen,
+    ButtonComponent: ButtonComponent,
+  }
+
+  const reopenButtonPluralProps = {
+    title: "Reopen incidents",
+    question: "Are you sure you want to reopen these incidents?",
+  }
+
+
   if (open) {
     return (
       <SignOffAction
-        dialogTitle="Manually close incident"
-        dialogContentText="Write a message describing why the incident was manually closed"
-        dialogSubmitText="Close now"
-        dialogCancelText="Cancel"
-        dialogButtonText={closeButtonText}
-        dialogInputLabel="Closing message"
-        isDialogInputRequired={false}
-        dialogInputType="text"
-        title="Manually close incident"
-        question="Are you sure you want to close this incident?"
-        onSubmit={onManualClose}
+        {...signOffActionDefaultProps}
+        {...(isBulk && signOffActionPluralProps)}
         {...signOffActionProps}
       />
     );
   } else {
     return (
       <ConfirmationButton
-        title={"Reopen incident"}
-        question={"Are you sure you want to reopen this incident?"}
-        onConfirm={onManualOpen}
-        ButtonComponent={ButtonComponent}
+        className={classes.dangerousButton}
         buttonProps={{
-            variant: "contained"
+            variant: "contained",
         }}
```

Comment:
When i tried to but `buttonProps` inside `reopenButtonDefaultProps` I got this error:

```
Type '{ children: string; color?: "inherit" | "primary" | "secondary" | "default" | undefined; disabled?: boolean | undefined; disableElevation?: boolean | undefined; disableFocusRipple?: boolean | undefined; ... 284 more ...; buttonProps: { ...; }; }' is not assignable to type '{ title: string; question: string; ButtonComponent?: "abbr" | "address" | "article" | "aside" | "b" | "bdi" | "bdo" | "big" | "blockquote" | "button" | "caption" | "cite" | "code" | "data" | ... 44 more ... | undefined; buttonProps?: Partial<...> | undefined; }'.
 The types of 'buttonProps.variant' are incompatible between these types.
  Type 'string' is not assignable to type '"text" | "outlined" | "contained" | undefined'.  TS2322
```

So i just left it where it was

Reply:
buttonProps is a type defined by MUI, but no worries we don't actually need to fix this